mixer/test/client/check_cache_hit/check_cache_hit_test.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2018 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checkCache
+
+import (
+	"fmt"
+	"testing"
+
+	mixerpb "istio.io/api/mixer/v1"
+	"istio.io/istio/mixer/test/client/env"
+)
+
+// Check attributes from a good GET request
+const checkAttributesOkGet = `
+{
+  "context.protocol": "http",
+  "mesh1.ip": "[1 1 1 1]",
+  "mesh2.ip": "[0 0 0 0 0 0 0 0 0 0 255 255 204 152 189 116]",
+  "mesh3.ip": "[0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 8]",
+  "request.host": "*",
+  "request.path": "/echo",
+  "request.time": "*",
+  "request.useragent": "Go-http-client/1.1",
+  "request.method": "GET",
+  "request.scheme": "http",
+  "source.uid": "POD11",
+  "source.namespace": "XYZ11",
+  "source.ip": "[127 0 0 1]",
+  "source.port": "*",
+  "target.name": "target-name",
+  "target.user": "target-user",
+  "target.uid": "POD222",
+  "target.namespace": "XYZ222",
+  "connection.mtls": false,
+  "request.headers": {
+    ":method": "GET",
+    ":path": "/echo",
+    ":authority": "*",
+    "x-forwarded-proto": "http",
+    "x-istio-attributes": "-",
+    "x-request-id": "*"
+  }
+}
+`
+
+// Report attributes from a good GET request
+var reportAttributesOkGet = [...]string{`
```

Comment:
please add comment on the different betwen [0] and [1]

Reply:
Done. Thanks.